cypress/integration/Pagalbinis.js

```diff
@@ -0,0 +1,38 @@
+ // Dictionary with roles and values - number of tabs it has
+ var roles = { "User":1, "Team Lead":2, "Manager":5, "Accountant":5, "Admin":6};
+  
+ //Mistake in the name
+ var name = "Saulius Lipkevičius"; 
+ var trueName = "Saulius Lipkevicius";
+
+ //Looping from 0 to 4 , overall 5 steps. 
+ for (let step = 0; step < Object.keys(roles).length ; step++) {  
+
+    describe('Sourcebooks login', function() {
+
+        it('Should display validation for empty user after attempted loggin', function () {
+            
+            cy.visit('/');
+            cy.get('.Select.not-valid').should('not.visible')
+            cy.get('[type="submit"]').click();
+            cy.get('.Select.not-valid').should('be.visible')
+        })
+
+        it('Should be able to login with role User', function () {
```

Comment:
It would be nice to replace 'User' with '${Object.keys(roles)[step]', so that the name of each test case represents the correct role